### DISCUSSION TITLE
SVT Play, högre än 960x540 går inte att välja

### DISCUSSION THREAD

**Discussion by PetJig** [stefansundin/privatkopiera#217] [General]
Varför kan jag bara välja upplösning 960x540 som högsta filstorlek under tillvalet Ström, på t.ex. sportevenemanget Skidskytte-VM?
Även andra äldre evenemang kan man bara välja som högst upplösning 960x540.
Spelar man upp evenemanget på SVT Plays egen sida så visas en högre upplösning.

Privatkopiera v0.4.1
i senast uppdaterad Google Chrome

**Comment by stefansundin** [comment]
Har du möjligtvis en länk till ett program med det här felet?

Det kan hända att det är löst i v0.4.2 nu när jag ändrat vilken ström som Privatkopiera använder som standard. Och nu kan du ändra på det själv i inställningarna också.

**Comment by PetJig** [comment]
Det här var ju ett år sedan, så jag har glömt specifikt vilka program det
var men det var många från skidskytte-vm 2024 som bara funkade med 960 som
högsta val. Kanske t.ex. denna:
https://www.svtplay.se/video/j4da7Yr/skidskytte-vm-1/damernas-jaktstart
Jag minns också då att många andra program inte gick att välja mer än 960,
t.ex. På Spåret.
Jag övergav då Privatkopiera och körde med svtplay-dl.exe istället, med det
så funkar/funkade det varje gång att få hem filer med högsta upplösning
även om den är betydligt långsammare. Jag hämtar aldrig hem program med
undertexter, så det är oväsentligt för mig.

Eftersom det var ett år sedan jag ställde frågan så kanske SVT har bytt
både servers och teknik sedan dess.

Mvh PetJig


Den mån 24 feb. 2025 kl 22:33 skrev Stefan Sundin ***@***.***
